dftimewolf/cli/dftimewolf_recipes.py

```diff
@@ -325,10 +325,9 @@ def ValidateArguments(self) -> None:
 
       switch = expanded_argument.switch.replace('--', '')
       argument_mandatory = switch == arg.switch
-      argument_set = switch in self.state.command_line_options
+      argument_value = self.state.command_line_options[switch]
```

Comment:
Maybe use .get() here to avoid key errors?

Reply:
Yes, definitely